attrs_strict/_commons.py

```diff
@@ -0,0 +1,15 @@
+def _is_newtype(type_):
+    """Checks if `type_` is :func:`typing.NewType`."""
```

Comment:
Again, I find this comment redundant, the name of the function **is** `_is_newtype`, it already defines what it does. Could you please remove it ?

Reply:
The function name should be descriptive but is not a full human sentence and never replaces documentation. Beside that the function name does not contain a reference to the type we want to check :func:`typing.NewType`. Since we do string comparison in the implementation, not even the function body contains a reference. Documenting source code is good coding practise and you shoud not encourage people in doing the opposite.